src/redis_index.c

```diff
@@ -33,8 +33,8 @@ void *InvertedIndex_RdbLoad(RedisModuleIO *rdb, int encver) {
     IndexBlock *blk = &idx->blocks[actualSize];
     blk->firstId = RedisModule_LoadUnsigned(rdb);
     blk->lastId = RedisModule_LoadUnsigned(rdb);
-    blk->numDocs = RedisModule_LoadUnsigned(rdb);
-    if (blk->numDocs > 0) {
+    blk->numEntries = RedisModule_LoadUnsigned(rdb);
+    if (blk->numEntries > 0) {
```

Comment:
Shouldn't we set numDocs here as well (when loading old rdb it should be the same no?)
Anyway we said we will drop support for old rdb so it does not really an issue..

Reply:
Ohh I see you drop numDocs entirely?

Reply:
For `IndexBlock` yes - replaced it with `numEntries`